.github/workflows/gh-ci.yaml

```diff
@@ -212,8 +212,16 @@ jobs:
     - name: deploy_docs
       if: github.event_name != 'pull_request'
       run: |
-        # place the deploy call here
-        echo "Oh, maple syrup roast parsnips [Richard Gowers]"
+        python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
+        cd package/MDAnalysis
+        export VERSION=$(python -c "import version; print(version.__version__)")
+        cd -
+        export GH_DOC_BRANCH=$GITHUB_REF
```

Comment:
not that it really matters (especially since I think you still need to do the `cd -` either way), but these could be all put inside an `env:` block if you happened to think it was cleaner that way.